### ISSUE TITLE
[BUG] nanographrag & ollama file indexing and embeddings error

### ISSUE THREAD

**Issue by scheckley** [Cinnamon/kotaemon#487]
### Description

installed kotaemon using the Docker method. I've set up the nano_graphrag as described in the readme.

Indexing error uploading pdf files:

```
INFO:httpx:HTTP Request: POST http://x.x.x.x/v1/embeddings "HTTP/1.1 500 Internal Server Error"
INFO:openai._base_client:Retrying request to /embeddings in 0.480649 seconds
```

I'm using ollama serving llama 3.2 and nomic-embed-text. All tests return fine, and I can get a response from ollama using curl to the embeddings api.  Simple queries are ok, but quality of results is poor. Embeddings don't seem to be working at all. I've had previous installations running better with the same embeddings settings but no graphrag.

I'm just wondering if anybody else has seen this error?  Thanks for any help.

### OS

Linux

### Additional information

Docker container deployed on Openshift.

**Comment by taprosoft** [comment]
Hi @scheckley please use the Test connection feature to test connection to Ollama server in Docker container.

https://github.com/Cinnamon/kotaemon/blob/main/docs/local_model.md#note

Connection to Ollama server in the host machine from Docker container can require a bit further configuration.

**Comment by scheckley** [comment]
is it possible to change chunk_size parameter? i see it referenced but can't find where it's set. thanks

**Comment by scheckley** [comment]
uploading and indexing a pdf with nano graphrag works fine:

![image](https://github.com/user-attachments/assets/9bf74e92-4dda-4136-b802-8b06ada0fe84)

but uploading the same pdf as a file collection causes error 500.  

![image](https://github.com/user-attachments/assets/b329bc07-59d8-44ac-8c2f-d27e898fa0eb)

I'll review the settings on the ollama side.

**Comment by scheckley** [comment]
corrected the problem by downgrading ollama to 0.3.14
